sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLog.scala

```diff
@@ -46,8 +46,7 @@ import org.apache.spark.sql.SQLContext
  * files in a directory always shows the latest files.
  */
 class HDFSMetadataLog[T: ClassTag](sqlContext: SQLContext, path: String)
-  extends MetadataLog[T]
-  with Logging {
+  extends MetadataLog[T] with Logging {
```

Comment:
Against the code style, don't you think? How to proceed with similar cases? Please guide.